Applications/Custom/mae_loss.h

```diff
@@ -23,7 +23,7 @@ namespace custom {
 
 /**
  * @brief A sample loss layer which calculates mean absolute error from output
- * @todo update this to LayerV1
+ * @todo update this to LayerV2
```

Comment:
do developers have to include layer_internal.h ?? or layer_devel.h? If this is the example for the developer, then I think it must be the layer_devel.h. Could you check this? And do we need to include manager.h? AFAK, we are not going to release manager.h.

Reply:
It is layer_devel, after the migration, this will be moving to including layer_devel :)